xtest/conftest.py

```diff
@@ -56,9 +65,46 @@ def pt_file(tmp_dir, size):
 
 
 @pytest.fixture
-def tmp_dir():
+def tmp_dir() -> str:
     dname = "tmp/"
     isExist = os.path.exists(dname)
     if not isExist:
         os.makedirs(dname)
     return dname
+
+
+@pytest.fixture
+def kas_public_keys(feature_flag: str) -> abac.PublicKey:
+    def load_cached_kas_keys() -> abac.PublicKey:
+        keyset: list[abac.KasPublicKey] = []
+        with open("../../platform/kas-cert.pem", "r") as rsaFile:
+            keyset.append(
+                abac.KasPublicKey(
+                    alg=abac.KAS_PUBLIC_KEY_ALG_ENUM_RSA_2048,
+                    kid="r1",
+                    pem=rsaFile.read(),
+                )
+            )
+        with open("../../platform/kas-ec-cert.pem", "r") as ecFile:
+            keyset.append(
+                abac.KasPublicKey(
+                    alg=abac.KAS_PUBLIC_KEY_ALG_ENUM_EC_SECP256R1,
+                    kid="e1",
+                    pem=ecFile.read(),
+                )
+            )
+        return abac.PublicKey(
+            cached=abac.KasPublicKeySet(
+                keys=keyset,
+            )
+        )
+
+    def load_local_kas_key() -> abac.PublicKey:
+        with open("../../platform/kas-cert.pem", "r") as rsaFile:
+            return abac.PublicKey(
+                local=rsaFile.read(),
+            )
+
+    if feature_flag == "cached_key":
```

Comment:
feature flag setup lgtm but can we make cached_key default since the java stuff is merged now?